sources/core/Stride.Core.CompilerServices/Analyzers/STRDIAG000AttributeContradiction.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Text;
+using Stride.Core.CompilerServices.Common;
+
+namespace Stride.Core.CompilerServices.Analyzers;
+[DiagnosticAnalyzer(LanguageNames.CSharp)]
+public class STRDIAG000AttributeContradiction : DiagnosticAnalyzer
+{
+    public const string DiagnosticId = "STRDIAG000";
+    private const string Title = "Inaccessible Member";
+    private const string MessageFormat = "There is an Attribute Contradiction on '{0}' Member. [DataMemberIgnoreAttribute] on a [DataMember] is not allowed. Except if it has also [DataMemberUpdatableAttribute] Attribute";
+    private const string Category = "CompilerServices";
```

Comment:
Category for most of these should be `Serialization`

Reply:
[07cceaf](https://github.com/stride3d/stride/pull/1864/commits/07cceafcfcde928ad3bb39e2efbdd5acd496b31c)

i think for all current errors its good to have serialization as category